runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/SmithingAction.java

```diff
@@ -116,7 +117,14 @@
 	STEEL_SQ_SHIELD(ItemID.STEEL_SQ_SHIELD, 38, 75),
 	STEEL_WARHAMMER(ItemID.STEEL_WARHAMMER, 39, 112.5f),
 	STEEL_BATTLEAXE(ItemID.STEEL_BATTLEAXE, 40, 112.5f),
-	GOLD_BAR_GOLDSMITH_GAUNTLETS(ItemID.GOLD_BAR, 40, 56.2f),
+	GOLD_BAR_GOLDSMITH_GAUNTLETS(ItemID.GOLD_BAR, 40, 56.2f)
```

Comment:
Speaking of related fixes, this could use a name override to differentiate it fro `GOLD_BAR`.